libs/sdk-model/src/theme/index.ts

```diff
@@ -0,0 +1,138 @@
+// (C) 2020 GoodData Corporation
+
+/**
+ * @public
```

Comment:
Is this API done? Because once we release this with @public tag, we cannot change it in a breaking way. I suggest using @beta for now.